its/projects/CSharp.Framework.4.8/Main/Common.cs

```diff
@@ -3,5 +3,6 @@
     public class Common
     {
         // FIXME: This line contains S1134 warning
+        public int Add(int a, int b) => a + b;
```

Comment:
The purpose of this is to avoid raising S2094 which is not available yet for the analyzer `v.8.51.0.59060` so it would complicate the assertions on `testCSharpFramework48`